src/recipe/test/e2e/createRecipe.controller.e2e-spec.ts

```diff
@@ -61,7 +62,7 @@ describe('Recipe Controller - Create', () => {
 
   beforeAll(async () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [RecipeModule, AuthModule],
+      imports: [RecipeModule, AuthModule, AppModule],
```

Comment:
I assume that `AppModule` imports `RecipeModule`, `AuthModule`
Do you need to import Recipe and Auth modules separately?